lib/utils/isnt-design-doc.js

```diff
@@ -0,0 +1,4 @@
+// Checks for a design doc, so we can filters out docs that shouldn't return in *All methods
+module.exports = function isntDesignDoc (row) {
+  return row.id.match(/^_design/) === null
```

Comment:
that would make a great follow-up PR 👍 do you want to send one? Thanks for checking it!